Shared/Samples/Show viewshed from point on map/ShowViewshedFromPointOnMapView.swift

```diff
@@ -0,0 +1,188 @@
+            // on the map.
+            let featureCollectionTable = FeatureCollectionTable(
+                fields: [Field](),
+                geometryType: Point.self,
+                spatialReference: spatialReference
+            )
+            // Creates a feature for the point tapped.
+            let feature = featureCollectionTable.makeFeature(geometry: point)
+            // Asynchronously adds that feature to the table.
+            try await featureCollectionTable.add(feature)
+            try await performGeoprocessing(featureCollectionTable)
+        }
+        
+        /// Contains the logic for the geoprocessing and passes the result on to another function to display.
+        /// - Parameter featureCollectionTable: Holds the tapped location feature
+        private func performGeoprocessing(_ featureCollectionTable: FeatureCollectionTable) async throws {
+            let params = GeoprocessingParameters(executionType: .synchronousExecute)
+            // Sets the parameters spatial reference to the point tapped on the map.
+            params.processSpatialReference = featureCollectionTable.spatialReference
+            params.outputSpatialReference = featureCollectionTable.spatialReference
+            // Create a feature with the feature collection table which has the reference to the tapped location.
+            let geoprocessingFeature = GeoprocessingFeatures(features: featureCollectionTable)
+            // Sets the observation point to the tapped location.
+            params.setInputValue(geoprocessingFeature, forKey: "Input_Observation_Point")
+            // Creates geoprocessing job and kicks off process of getting viewshed for the point.
+            geoprocessingJob = geoprocessingTask.makeJob(parameters: params)
+            geoprocessingJob?.start()
+            // Get the result of the geoprocessing job asynchronously.
+            let output = try await geoprocessingJob?.output
+            if let resultFeatures = output?.outputs["Viewshed_Result"] as? GeoprocessingFeatures,
+               let featureSet = resultFeatures.features {
+                processFeatures(in: featureSet)
+            }
+        }
+        
+        /// If the feature set is returned from the geoprocessing, it iterates through each feature and adds it to the
+        /// graphic overlay to display to the user.
+        /// - Parameter resultFeatures: Passes on the results of the geoprocessing so that they can be displayed.
+        private func processFeatures(in featureSet: FeatureSet) {
+            // Iterates through the feature set.
+            for feature in featureSet.features() {
+                // Creates the graphic for each feature's geometry.
+                let graphic = Graphic(geometry: feature.geometry)
+                // Sets the graphic on the overlay to display to the user.
+                resultGraphicsOverlay.addGraphic(graphic)
+            }
+        }
+    }
```

Comment:
No need to changed, can also be simplified into

```swift
resultGraphicsOverlay.addGraphics(featureSet.features().map { Graphic(geometry: $0.geometry) })
```

Reply:
since it's a one liner, would it need to be in its own function?

Reply:
If you make this change, then no need to be in a separate method.

Reply:
Okay, yeah I added it because I was updating it for the other suggestions.